pkg/skaffold/event/event.go

```diff
@@ -712,3 +712,14 @@ func BuildSequenceFailed(err error) {
 	handler.state.BuildState.StatusCode = aiErr.ErrCode
 	handler.stateLock.Unlock()
 }
+
+func SkaffoldInitFailed(err error) {
```

Comment:
This name risks confusion with `skaffold init`.  `InititializationFailed` is better, and is in keeping with the corresponding error codes.